### ISSUE TITLE
Tinymce/bbcode 

### ISSUE THREAD

**Issue by tgtje** [e107inc/e107#614]
USING  news create OR Page (files age : latest 5 hours ago) 2-6
INPUT AREA (window : body)

Upgrade of v1 to v2 : succeeded with 1 solvable issue : new issue made/making> forum related issue =upgrade )

Creating news and/or pages

(as installed = tinymce4 Upgraded site
trying to add youtube etc. 
When selecting youtube it opens media manager news : youtube (2 movies); not selectable
Adding an image : media m.. displayed : not selectable

adding youtube embedding code ( i frame ) works, txt works. Trying to use any image or related >  not selectable

Here are some error reports
Tijdstempel: 2-6-2014 14:44:05
Fout: TypeError: $(...).tooltip is not a function
Bronbestand: http://domain/e107_web/js/core/admin.jquery.js?1401712003
Regel: 76 =      $(this).tooltip({opacity:1.0,fade:true, placement: pos});

When creating news selecting an image at bottom (first block)
it opens mediamanager.Selecting an image shown. marks it, but does NOT close the dialog : no file selection (closing window)

Tijdstempel: 2-6-2014 14:52:14
Fout: TypeError: parent.document.getElementById(...) is null
Bronbestand: http://domain/e107_admin/image.php?mode=main&action=dialog&for=news&tagid=news-thumbnail-0&iframe=1&video=1
Regel: 1

Tijdstempel: 2-6-2014 14:56:25
Fout: TypeError: e is undefined
Bronbestand: http://tinymce.cachefly.net/4.0/tinymce.min.js?1401712003
Regel: 9

Tijdstempel: 2-6-2014 15:02:11
Fout: TypeError: $(...).draggable is not a function
Bronbestand: http://domain/e107_web/js/core/admin.jquery.js?1401712003
Regel: 82

Uninstalled Tinymce4
Installed Tinymce

In BOTH cases i have NO possibilties to add media related
Preferences adjusting (disabling wysiwig) does NOT change display of window or makes changes to method for input

Tijdstempel: 2-6-2014 15:04:47
Fout: TypeError: tinymce is null
Bronbestand: http://domain/e107_plugins/tinymce/tiny_mce_popup.js?1401712003
Regel: 5

 basically i can not create a page or news post WITH media
